### PR TITLE
Fix: release script to include package.json

### PR DESCRIPTION
**Pull request by JulienBlacas**

### CODE REVIEW

File: scripts/create-release.sh [hunk 1]
```diff
@@ -7,11 +7,14 @@ git fetch origin
 git checkout origin/feature/sparc-eq-derivatives
```
Comment (matthieucrouzet-sym) [inline scripts/create-release.sh:7]: shouldn't we remove this line and the one above if we say that the user as to checkout the branch on which one he want the release

File: README.md [hunk 2]
```diff
@@ -94,5 +94,7 @@ import { Box , Text, Button } from 'symphony-bdk-ui-toolkit';
 ```
 
 ## How create a release
-Open a terminal in at the root of your git
-And run `yarn create-release [patch|minor|major] "your commit message"`
\ No newline at end of file
+Once your PR is merged to the destination branch (i.e `features/sparc-eq-derivatives`)
+1) Checkout the branch
```
Comment (matthieucrouzet-sym) [inline README.md:98]: do git fetch before checkout to the branch

File: scripts/create-release.sh (hunk 1 above)
Comment (JulienBlacas) [reply scripts/create-release.sh]: @matthieucrouzet-sym Yes you are right !

**Review by matthieucrouzet-sym — APPROVED** [review]
lgtm

**Pull request merged**